tests/python/frontend/tflite/test_forward.py

```diff
@@ -144,8 +143,6 @@ def compare_tflite_with_tvm(in_data, in_name, input_tensors,
             for i in range(len(tflite_output)):
                 tvm.testing.assert_allclose(tflite_output[i], tvm_output[i], atol=1e-5, rtol=1e-5)
 
-        sess.close()
```

Comment:
line 127 opens new `with tf.Session() as sess:` context block.
After python finishes `with tf.Session` block execution it calls`sess.__exit__()` method automatically which calls `self.close()` to close the session.
We do not need to close `sess` explicitly.